diff --git a/cmake/modules/FindLibRaw.cmake b/cmake/modules/FindLibRaw.cmake
index c6187d9..e33b06b 100644
--- a/cmake/modules/FindLibRaw.cmake
+++ b/cmake/modules/FindLibRaw.cmake
@@ -77,3 +77,21 @@ MARK_AS_ADVANCED(LibRaw_VERSION_STRING
                  LibRaw_DEFINITIONS
                  LibRaw_r_DEFINITIONS
                  )
+
+IF(LibRaw_FOUND AND NOT TARGET LibRaw::LibRaw)
+   add_library(LibRaw::LibRaw INTERFACE IMPORTED)
+   set_target_properties(LibRaw::LibRaw PROPERTIES
+      INTERFACE_INCLUDE_DIRECTORIES "${LibRaw_INCLUDE_DIR}"
+      INTERFACE_LINK_LIBRARIES "${LibRaw_LIBRARY_DEBUG}"
+      # INTERFACE_COMPILE_DEFINITIONS "${LibRaw_DEFINITIONS}"
+   )
+ENDIF()
+
+# IF(LibRaw_FOUND AND NOT TARGET LibRaw::LibRaw_r)
+#    add_library(LibRaw::LibRaw_r INTERFACE IMPORTED)
+#    set_target_properties(LibRaw::LibRaw_r PROPERTIES
+#       INTERFACE_INCLUDE_DIRECTORIES "${LibRaw_INCLUDE_DIR}"
+#       INTERFACE_LINK_LIBRARIES "${LibRaw_r_LIBRARIES}"
+#       # INTERFACE_COMPILE_DEFINITIONS "${LibRaw_r_DEFINITIONS}"
+#    )
+# ENDIF()
